src/FirePokemon.java

```diff
@@ -0,0 +1,13 @@
+public class FirePokemon extends Pokemon {
+
+    private String flamethrower = "flamethrower";
+
+    public FirePokemon(String name, int level){
+        super(name, level);
+        super.setType("fire");
+    }
+
+    public void flamethrower() {
+        System.out.println("Pokemon throws flame");
```

Comment:
deze printstatement was een goed moment geweest om "this.getName()" te gebruiken in plaats van "Pokemon"